jackett_launcher.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Helper script to fix
+# https://github.com/Jackett/Jackett/issues/5208#issuecomment-547565515
+
+JACKETT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
```

Comment:
I know this is a fringe use-case, but on a multi-user system, to prevent future issues, it may be best to define the user and pgrep only processes from that user (to avoid searching for processes that don't belong to the user running the service and potentially finding matches that don't apply to our instance):

```
# Get user running the service
USER=$(whoami)

.
.
.
while pgrep -u ${USER} JackettUpdater > /dev/null ; do
     sleep 1
done
```

Reply:
done